{"uuid":"b451636e-3ebe-4354-8c1a-6474059c8002","historyId":"6450da269d4a6208d3b70b1431af3986","testCaseId":"[engine:junit-jupiter]/[class:de.qytera.QForAutomation.QForAutomationApplicationTests]/[method:contextLoads()]","testCaseName":"contextLoads()","fullName":"de.qytera.QForAutomation.QForAutomationApplicationTests.contextLoads","labels":[{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:de.qytera.QForAutomation.QForAutomationApplicationTests]/[method:contextLoads()]"},{"name":"host","value":"qytera128"},{"name":"thread","value":"30856@qytera128.main(1)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"de.qytera.QForAutomation.QForAutomationApplicationTests"},{"name":"testClass","value":"de.qytera.QForAutomation.QForAutomationApplicationTests"},{"name":"testMethod","value":"contextLoads"},{"name":"suite","value":"de.qytera.QForAutomation.QForAutomationApplicationTests"}],"links":[],"name":"contextLoads()","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'QYTERA128', ip: '192.168.178.144'\nBuild info: version: '4.19.1', revision: 'abe0ee07dc'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.3'\nDriver info: org.openqa.selenium.remote.RemoteWebDriver\nCommand: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--headless, --disable-gpu, --remote-allow-origins=*], extensions: []}}]}]\nCapabilities {browserName: chrome, goog:chromeOptions: {args: [--headless, --disable-gpu, --remote-allow-origins=*], extensions: []}}","trace":"org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. \nHost info: host: 'QYTERA128', ip: '192.168.178.144'\nBuild info: version: '4.19.1', revision: 'abe0ee07dc'\nSystem info: os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.3'\nDriver info: org.openqa.selenium.remote.RemoteWebDriver\nCommand: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--headless, --disable-gpu, --remote-allow-origins=*], extensions: []}}]}]\nCapabilities {browserName: chrome, goog:chromeOptions: {args: [--headless, --disable-gpu, --remote-allow-origins=*], extensions: []}}\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:536)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:232)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:161)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:141)\r\n\tat de.qytera.QForAutomation.QForAutomationApplicationTests.contextLoads(QForAutomationApplicationTests.java:99)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\nCaused by: java.io.UncheckedIOException: java.net.ConnectException\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute0(JdkHttpClient.java:436)\r\n\tat org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)\r\n\tat org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)\r\n\tat org.openqa.selenium.remote.http.jdk.JdkHttpClient.execute(JdkHttpClient.java:352)\r\n\tat org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:54)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:89)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:75)\r\n\tat org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:61)\r\n\tat org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:162)\r\n\tat org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)\r\n\tat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:518)\r\n\t... 7 more\r\nCaused by: java.net.ConnectException\r\n\tat java.net.http/jdk.internal.net.http.common.Utils.toConnectException(Utils.java:1047)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:198)\r\n\tat java.net.http/jdk.internal.net.http.Http1Exchange.sendHeadersAsync(Http1Exchange.java:239)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.lambda$responseAsyncImpl0$9(Exchange.java:472)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.checkFor407(Exchange.java:404)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.lambda$responseAsyncImpl0$10(Exchange.java:476)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandleStage(CompletableFuture.java:950)\r\n\tat java.base/java.util.concurrent.CompletableFuture.handle(CompletableFuture.java:2340)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl0(Exchange.java:476)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.responseAsyncImpl(Exchange.java:380)\r\n\tat java.net.http/jdk.internal.net.http.Exchange.responseAsync(Exchange.java:372)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:408)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsyncImpl$7(MultiExchange.java:449)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandle(CompletableFuture.java:934)\r\n\tat java.base/java.util.concurrent.CompletableFuture.uniHandleStage(CompletableFuture.java:950)\r\n\tat java.base/java.util.concurrent.CompletableFuture.handle(CompletableFuture.java:2340)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.responseAsyncImpl(MultiExchange.java:439)\r\n\tat java.net.http/jdk.internal.net.http.MultiExchange.lambda$responseAsync0$2(MultiExchange.java:341)\r\n\tat java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)\r\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\r\n\tat java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1773)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:833)\r\nCaused by: java.nio.channels.UnresolvedAddressException\r\n\tat java.base/sun.nio.ch.Net.checkAddress(Net.java:149)\r\n\tat java.base/sun.nio.ch.Net.checkAddress(Net.java:157)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.checkRemote(SocketChannelImpl.java:816)\r\n\tat java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:839)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.lambda$connectAsync$0(PlainHttpConnection.java:183)\r\n\tat java.base/java.security.AccessController.doPrivileged(AccessController.java:569)\r\n\tat java.net.http/jdk.internal.net.http.PlainHttpConnection.connectAsync(PlainHttpConnection.java:185)\r\n\t... 23 more\r\n"},"stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1718374368894,"stop":1718374374291}